cache-config/testing/docker/trafficserver/run.sh

```diff
@@ -18,32 +18,58 @@
 # under the License.
 #
 
-function initBuildArea() {
-  cd /root
-
-  # prep build environment
-  [ -e rpmbuild ] && rm -rf rpmbuild
-  [ ! -e rpmbuild ] || { echo "Failed to clean up rpm build directory 'rpmbuild': $?" >&2; exit 1; }
-  mkdir -p rpmbuild/{BUILD,BUILDROOT,RPMS,SPECS,SOURCES,SRPMS} || { echo "Failed to create build directory '$RPMBUILD': $?" >&2;
-  exit 1; }
+function die() {
+  { test -n "$@" && echo "$@"; exit 1; } >&2 
 }
 
-setowner() {
-	own="$(stat -c%u:%g "$1")"
-	shift
-	chown -R "${own}" "$@"
-}
-trap 'exit_code=$?; setowner /trafficcontrol /trafficcontrol/dist; exit $exit_code' EXIT;
-
-case ${ATS_VERSION:0:1} in
-  8) cp /trafficserver-8.spec /trafficserver.spec
-     ;;
-  9) cp /trafficserver-9.spec /trafficserver.spec
-     ;;
-  *) echo "Unknown trafficserver version was specified"
-     exit 1
-     ;;
-esac
+# sets minimum defaults if these are undefined. These are normally defined in the
+# environment section of the GHA workflow see, '.github/workflows/cache-config-tests.yml'
+RHEL_VERSION="${RHEL_VERSION:-8}"
+ATS_VERSION="${ATS_VERSION:-8.1.x}"
+
+echo "RHEL_VERSION:${RHEL_VERSION}"
+echo "ATS_VERSION:${ATS_VERSION}"
+
+mkdir -p /opt/build
+cd /opt/build
+
+# build openssl 1.1.1 if RHEL_VERSION is not 8 or greater.
+if [[ ${RHEL_VERSION%%.*} -le 7 ]]; then
+  git clone $OPENSSL_URL --branch $OPENSSL_TAG || die "Failed to fetch the OpenSSL Source."
+  (
+    cd /opt/build/openssl && \
+    ./config --prefix=/opt/trafficserver/openssl --openssldir=/opt/trafficserver zlib && \
+    make -j$(nproc) && \
```

Comment:
No need to include `\` after `&&` outside of Dockerfiles.

Reply:
I had them in there because I think they improve code readability but, I'll remove them.